src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -105,6 +113,9 @@ export class MatChipGrid extends _MatChipGridMixinBase implements AfterContentIn
   /** Subscription to blur changes in the chips. */
   private _chipBlurSubscription: Subscription | null;
 
+  /** Subscription to focus changes in the chips. */
+  private _chipFocusSubscription: Subscription | null;
```

Comment:
I think it's done this way because we drop all the subscriptions and resubscribe whenever the chips in the grid change, not just when the grid is destroyed. 

https://github.com/angular/components/blob/master/src/material/chips/chip-list.ts#L377

Reply:
Got it, that makes sense